### ISSUE TITLE
"All Arguments" reference is incorrect

### ISSUE THREAD

**Issue by bknittel** [MicrosoftDocs/windowsserverdocs#922]
The page says that %\* (percent, backslash, asterisk) is replaced with all arguments, but this is incorrect. The sequence is %*  (percent asterisk). No backslash. Maybe this was just an error in transcribing the help content for this web page.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 144631f0-4625-641e-7bb5-63a5d0f49f27
* Version Independent ID: 8db973db-7eeb-acc6-0f1b-db598a8de43b
* Content: [call](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/call)
* Content Source: [WindowsServerDocs/administration/windows-commands/call.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/master/WindowsServerDocs/administration/windows-commands/call.md)
* Product: **windows-server-threshold**
* GitHub Login: @coreyp-at-msft
* Microsoft Alias: **coreyp**

**Comment by lizap** [comment]
You're right - this was a problem with our source tagging in the article. We'll get it cleaned up. Thanks!

**Comment by bknittel** [comment]
Sure thing. I guess, for your $7.5B, they threw in a few extra backslashes
for free.
